vaccine_feed_ingest/runners/nv/immunizenevada_org/normalize.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import pathlib
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest.schema import schema
+from vaccine_feed_ingest.utils.normalize import provider_id_from_name
+
+
+def _get_id(site: dict) -> str:
+    return f"immunizenevada_org:{site['id']}"
+
+
+def _get_address(address: str) -> schema.Address:
+    parts = address.split(",")
+    street2 = None
+    if len(parts) == 5:
+        street2 = parts[1]
+
+    result = schema.Address(
+        street1=parts[0],
+        street2=street2,
+        city=parts[-3],
+        state=parts[-2],
+        zip=parts[-1],
+    )
+    return result
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+    contacts = None
+
+    phone = None
+    if "contact-phone" in site:
+        phone = site["contact-phone"]
+
+    # Filter out sites with a url of "/"
+    website = None
+    if site["url"].startswith("http"):
+        website = site["url"]
+
+    if phone or website:
+        general_contact = schema.Contact(
+            contact_type="general",
+            phone=phone,
+            website=website,
+        )
+        contacts = [general_contact]
+
+    return contacts
+
+
+def _get_links(site: dict) -> Optional[List[schema.Link]]:
+    links = []
+    immunize_nv_link = schema.Link(
+        authority="immunizenevada_org",
+        id=site["title"],
+    )
+    links.append(immunize_nv_link)
+
+    parsed_provider_link = provider_id_from_name(site["title"])
+    if parsed_provider_link is not None:
+        provider_link = schema.Link(
+            authority=parsed_provider_link[0],
+            id=parsed_provider_link[1],
+        )
+        links.append(provider_link)
+
+    return links
+
+
+def normalize(site: dict, timestamp: str) -> schema.NormalizedLocation:
+    return schema.NormalizedLocation(
+        id=_get_id(site),
+        name=site["title"],
```

Comment:
It looks like a few of these records include an age restriction in the title, which will make them challenging for us to match later (e.g., `CVS 2990 (16+)`). Do you see an easy way to strip this out?

Reply:
@eliblock The NV data only contains "(16+)" as an age restriction in the title, so this regex would work. And it's  specific enough to allow titles like `CVS (1234)` to pass through unaltered.
```
title = re.sub(r"\([0-9]+\+\)", "", title)
```